src/UI/Icon.elm

```diff
@@ -1,32 +1,57 @@
 module UI.Icon exposing
-    ( Icon
-    , add
-    , backwardContent
-    , close
-    , edit
-    , eventLog
+    ( svgSpriteImport
+    , Icon, add, close, edit, eventLog, previousContent, logout, notifications, paackSpaces, packages, print, nextContent, sandwichMenu, search, seeMore, toggle, toggleDown, toggleUp, warning
     , getHint
-    , leftArrow
-    , logout
-    , notifications
-    , paackSpaces
-    , packages
-    , print
-    , renderElement
-    , rightArrow
-    , sandwichMenu
-    , search
-    , seeMore
-    , svgSpriteImport
-    , toggle
-    , toggleDown
-    , toggleUp
-    , warning
     , withColor
-    , withCustomSize
-    , withSize
+    , withSize, withCustomSize
+    , renderElement
     )
 
+{-| `UI.Icon` is an implementation of icons using an SVG-spreadsheet.
+
+Icons' names are after their proposed action/symbolism instead of describing its shape.
+Consequently, we don't use the same icon for different operations as these would confuse the end-users.
+
+To use these icons, first, you must insert this spreadsheet once (and only once) in the layout.
```

Comment:
I think you want to say sprite sheet here? but anyway we could rephrase do to something like:

"... you must insert this `Html` in the root of your view, as this function injects a custom element with the SVG sprite sheet"

something in that line, because I think it's important to mention that is a custom element behind the scenes ( it was just an example, I'm sure you can find a better explanation then mine :D )

Reply:
See if it's better now!